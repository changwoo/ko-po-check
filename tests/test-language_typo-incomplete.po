# -*- text -*-
# language/typo2bul
#
msgid ""
msgstr ""
"Project-Id-Version: test 17\n"
"POT-Creation-Date: 2010-03-06 11:02+0900\n"
"PO-Revision-Date: 2010-03-06 11:02+0900\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

msgid "blahblah"
msgstr "ㅂ르라블라"

msgid "foooh"
msgstr "푸ㅜ"
