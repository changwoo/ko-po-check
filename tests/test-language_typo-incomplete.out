-:16: "ㅂ": 음절이 불완전합니다.  오타로 보입니다
-:19: "ㅜ": 음절이 불완전합니다.  오타로 보입니다
